src/actions/index.js

```diff
@@ -1,25 +1,51 @@
+import { geocodeGoogle, breezoMeter } from "../api";
+import { saveToHistory } from "../utilities";
+
 export const RECEIVE_LOCATION = 'RECEIVE_LOCATION';
 
-export function receiveLocation(location, json) {
+export function receiveLocation(newLocation) {
   //dispatching an action to the store
   return {
     type: 'RECEIVE_LOCATION',
-    date: json.datetime,
-    name: location,
-    aqi: json.breezometer_aqi,
-    description: json.breezometer_description
+    date: newLocation.date,
+    name: newLocation.name,
+    aqi: newLocation.aqi,
+    description: newLocation.description,
+    color: newLocation.color
   };
 };
 
 export const fetchLocation = location => {
```

Comment:
you should be consistent with how you define functions, above you use `function` syntax whereas here you are assigning an anonymous function to a `const`